.npmignore

```diff
@@ -0,0 +1,6 @@
+images
```

Comment:
So after it's been debating a few different places, I think the "standard" is:

```
*
!lib/**
```

Check https://github.com/hapijs/good/blob/master/.npmignore as a reference.